model/tensorflow/tests/test_dnnr.py

```diff
@@ -4,17 +4,18 @@
 from typing import Type
 
 import numpy as np
-from dffml.repo import Repo, RepoData
+from dffml_model_tensorflow.dnnr import (
```

Comment:
Imports from plugin (`dffml_model_scratch`) should go after imports from `dffml`